app/_src/production/cp-deployment/multi-zone.md

```diff
@@ -38,12 +38,12 @@ Find the external IP and port of the `{{site.mesh_cp_zone_sync_name_prefix}}glob
 
 ```sh
 kubectl get services -n {{site.mesh_namespace}}
-NAMESPACE     NAME                   TYPE           CLUSTER-IP      EXTERNAL-IP      PORT(S)                                                                  AGE
-{{site.mesh_namespace}}   {{site.mesh_cp_zone_sync_name_prefix}}global-zone-sync     LoadBalancer   10.105.9.10     35.226.196.103   5685:30685/TCP                                                           89s
-{{site.mesh_namespace}}   {{site.mesh_cp_name}}     ClusterIP      10.105.12.133   <none>           5681/TCP,443/TCP,5676/TCP,5677/TCP,5678/TCP,5679/TCP,5682/TCP,5653/UDP   90s
+# NAMESPACE     NAME                   TYPE           CLUSTER-IP      EXTERNAL-IP      PORT(S)                                                                  AGE
```

Comment:
Should we maybe prefix the first line with `$ ` instead?

Reply:
It would be harder to just copy the command then, but I've put the result of commands in separate text blocks below the commands, so should be better now

Reply:
Is it harder to copy them? Most documentation uses the $ prefix to indicate a command, doesn't it?

Reply:
But it doesn't mean it's a great UX, when you have a button to copy the content of the code block and if you use it then after pasting the command in your terminal, you have to manually remove `$` and in our example the result of the command which was present in the same code block.

Reply:
Yeah I think this is just a buggy copy button then